engine/src/board.rs

```diff
@@ -358,10 +397,12 @@ impl Board {
     }
 
     pub fn spawns_left(&self, color: &Color, game_type: GameType) -> bool {
-        let has_reserve_bugs = self.reserve(color, game_type)
+        let has_reserve_bugs = self
+            .reserve(color, game_type)
             .iter()
             .filter(|(_, v)| **v > 0)
-            .count() > 0;
+            .count()
```

Comment:
nit: `.count() > 0;` looks more readable to me, but happy to defer to `cargo fmt` if it thinks otherwise

Reply:
This was `cargo fmt` :grimacing: